plugins/modules/zos_backup_restore.py

```diff
@@ -186,7 +186,7 @@
   hlq:
     description:
       - Specifies the new HLQ to use for the data sets being restored.
-      - Defaults to running user's username.
+      - Defaults to running original HLQ.
     type: str
     required: false
   tmp_hlq:
```

Comment:
edit the description of the default hlq behavior here too, it still says the default is Ansible user

>      - The default HLQ is the Ansible user that executes the module and if
>        that is not available, then the value of C(TMPHLQ) is used.